appjp/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+const App: React.FC = () => {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <header2> Big ol balls </header2>
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Hello World
```

Comment:
Here is a comment. I will bitch at you about things and then you can fix it.

- [ ] Acknowledge this task and then I'll approve your pr